CHANGELOG.md

```diff
@@ -1,10 +1,51 @@
 # Changelog
 
-## main / unreleased
-
-* [BUGFIX] OTLP receiver: Generate `target_info` samples between the earliest and latest samples per resource. #16737
-* [BUGFIX] Config: Infer escaping scheme when scrape config validation scheme is set.
+## 3.7.0-rc.0 / 2025-10-02
+
+* [FEATURE] PromQL: Add support for experimental anchored and smoothed rate. #16457
+* [FEATURE] Federation: NHCB support in federation. #17215
+* [FEATURE] PromQL: Add `first_over_time(...)` and `ts_of_first_over_time(...)` behind feature flag `experimental-promql-functions #16963 #17021
+* [FEATURE] Remote-write: Add support for Azure Workload Identity as an authentication method for the receiver. #16788
+* [FEATURE] Remote-write: Add type and unit labels to outgoing time series in remote-write 2.0 when the `type-and-unit-labels` feature flag is enabled. #17033
+* [FEATURE] TSDB: Track stale series in memory based on stale samples. Exposed via `prometheus_tsdb_head_stale_series`. #16925
```

Comment:
This was released in 3.6

Reply:
I searched for PRs merged after august 1, but this was picked onto release-3.6 later :facepalm: 
<img width="498" height="173" alt="image" src="https://github.com/user-attachments/assets/49328ec1-68e8-4471-8871-a762cc06cc57" />